src/stream_base.cc

```diff
@@ -293,17 +300,20 @@ int StreamBase::WriteString(const FunctionCallbackInfo<Value>& args) {
 
 void StreamBase::CallJSOnreadMethod(ssize_t nread,
                                     Local<ArrayBuffer> ab,
-                                    size_t offset) {
+                                    size_t offset,
+                                    bool skip_nread_checks) {
```

Comment:
Can you use an enum for this? That's more self-descriptive at call sites.

Reply:
You mean an enum with only two elements or some kind of 'flags'-like enum?

Reply:
The former, `enum { DONT_SKIP_NREAD_CHECKS, SKIP_NREAD_CHECKS }`.

Reply:
Added.